src/shell/shell.cpp

```diff
@@ -668,7 +668,20 @@ void SHELL_Init() {
 	        "Type CD drive: to display the current directory in the specified drive.\n"
 	        "Type CD without parameters to display the current drive and directory.\n");
 	MSG_Add("SHELL_CMD_CLS_HELP","Clear screen.\n");
-	MSG_Add("SHELL_CMD_DIR_HELP","Directory View.\n");
+	MSG_Add("SHELL_CMD_DIR_HELP","Displays a list of files and subdirectories in a directory.\n");
+	MSG_Add("SHELL_CMD_DIR_HELP_LONG","DIR [drive:][path][filename] [/[W|B]] [/P] [/[AD]|[A-D]] [/O[-][N|E|S|D]]\n\n"
```

Comment:
Very nice description :) aside of wording changes I mentioned in other comments - please fix formatting of this line.

Our coding convention calls for using tabs for indentation, but spaces for continuation. So for example this means line 672 should start with 1 tab (it's ok already), but line 673 should start with 1 tab followed by 8 spaces. This will make the code look consistent no matter the size of tab configured in the user's editor. Following this rule might be tricky, that's why `clang-format` will deal with this and format this statement correctly.

For example - in my development env (vim+clang-format) I would simply place cursor in line 672 and press Ctrl+K - and voila - the whole statement formatted perfectly :)

clang-format can be integrated with about every code editor out there; and even if you're using some more obscure one, we have script: `scripts/format-commit.sh` to help with formatting changes made in the latest commit.

Reply:
Done.